flygym/examples/vision_connectome_model/viz.py

```diff
@@ -24,16 +24,14 @@ def visualize_vision(
     video_path: Path,
     retina: Retina,
     retina_mapper: RetinaMapper,
-    rendered_image_hist: List[np.ndarray],
-    vision_observation_hist: List[np.ndarray],
-    nn_activities_hist: List[np.ndarray],
-    focus_masks_hist: List[np.ndarray],
+    viz_data_all: Tuple[Dict[str, np.ndarray]],
```

Comment:
changes to `visualize_vision` break `response_to_fly.py` as it used the old call signature of `visualize_vision`